include/inexor/vulkan-renderer/mesh_buffer.hpp

```diff
@@ -17,19 +21,67 @@ namespace inexor::vulkan_renderer {
 /// used during the same render operations, provided that their data is refreshed,
```

Comment:
This is the comment block ?

```
/// @brief A structure which bundles vertex buffer and index buffer (if existent).
/// It contains all data which are related to memory allocations for these buffers.
/// @todo Driver developers recommend that you store multiple
/// buffers, like the vertex and index buffer, into a single VkBuffer and use offsets
/// in commands like vkCmdBindVertexBuffers. The advantage is that your data
/// is more cache friendly in that case, because it’s closer together. It is even possible
/// to reuse the same chunk of memory for multiple resources if they are not
/// used during the same render operations, provided that their data is refreshed,
/// of course. This is known as aliasing and some Vulkan functions have explicit
/// flags to specify that you want to do this.
```

Reply:
I was speaking about the capitalization

Reply:
but it's in the middle of the sentence?